src/Components/Firestore.js

```diff
@@ -211,35 +208,105 @@ export function generateId() {
   return doc(collection(db, "test")).id;
 }
 
+export async function getReviewReactions(uid, uniqueEntityId) {
+  let reactionsRef = doc(db, "users", uid, "reactions", uniqueEntityId);
+  let reactionSnap = await getDoc(reactionsRef);
+  // Use default value if the document doesn't exist yet.
+  if (!reactionSnap.data()) {
+    return {
+      uniqueEntityId: uniqueEntityId,
+      applause: false,
+      heart: false,
+      trash: false,
+    };
+  } else {
+    return reactionSnap.data();
+  }
+}
+
 // Handle "watchlistData / reactions" on firestore
 
-export async function GetListReactions(docId, setListRxns) {
-  try {
-    let animeRef = doc(db, "watchlistData", docId, "reactions", "emojis");
-    let querySnapshot = await getDoc(animeRef);
+export async function getUserReactions(uid, docId) {
```

Comment:
Is this function the same as `getReviewReactions` above?  If so, do we need both, or should we consolidate to just one?  If so, let's use the generally-named `getUserReactions`

Reply:
Great catch!! Functions have been consolidated.